project/Dependencies.scala

```diff
@@ -3,12 +3,14 @@ import sbt._
 object Dependencies {
 
   object V {
+    val java8Compat = "0.9.1"
+
     val cats       = "2.1.1"
     val catsEffect = "2.1.4"
     val fs2        = "2.4.2"
     val munit      = "0.7.11"
     val newtype    = "0.4.4"
-    val pulsar     = "2.5.2"
+    val pulsar     = "2.6.0"
```

Comment:
Do we want to update the version yet? I think we are still running Pulsar 2.5.x

Reply:
Actually yes, we didn't migrate it yet. I'll fix this